config.sample.yml

```diff
@@ -80,6 +80,10 @@ generic:
   userIdPrefix: _webhooks_
   allowJsTransformationFunctions: false
   waitForComplete: false
+feeds:
+  # (Optional) Support for RSS/Atom feeds.
+  enabled: false
+  pollIntervalSeconds: 600
```

Comment:
it would be awesome if this was configurable on a per-feed basis, up (down?) to an admin configured minimum

Reply:
I've pondered it but I feel like that has more edgecases than usecases :)

The `pollIntervalSeconds` is something that the server admin would set to strike a balance between the network traffic on their server (and how "nice" they are to feed hosts) and the responsiveness of the subscriptions.

If we treat it as a minimum interval, we're opening an option for individual subscribers to say "I want to be nicer than this" – which is valid if they don't care about responsiveness that much, but it only works if all subscribers to this feed decide to be nice – it only takes one eager subscriber to force us to refresh it at a minimum anyway.

If we treat it as a maximum on the other hand, any user setting it up can effectively break the contract that the server admin signed, so it's not something we'd want.

So we could treat it as a default and allow a range of intervals to use, but it seriously feels like it's more hassle than it's worth: configuration-, UX- and complexity-wise. Is there good usecase for this that I didn't think of? I'm happy to change my mind :)

Reply:
You make a compelling argument, this might be unnecessary as a feature after all.